tsconfig.json

```diff
@@ -1,75 +1,45 @@
 {
   "compilerOptions": {
-    /* Visit https://aka.ms/tsconfig.json to read more about this file */
```

Comment:
Kan være lurt å ha med denne lenka i toppen for å enkelt finne til dokumentasjon for formatet.